content/news/buzz/project-spotlight-colosseum/contents+es.lr

```diff
@@ -0,0 +1,33 @@
+﻿title: Proyecto Spotlight: Colosseum
+---
+pub_date: 2017-10-06
+---
+author: Russell Keith-Magee
+---
+body:
+
+*Este artículo fue publicado originalmente en la lista de correo de Entusiastas BeeWare. Si desea recibir actualizaciones periódicas sobre el proyecto BeeWare, ¿por qué no* `suscribirse`_?
```

Comment:
I think we are using Tu instead of Usted in all the Spanish translations.

`Si desea` - > `Si deseas`
`¿por qué no* `suscribirse`_?` -> ¿Por qué no* `suscribirse`_?